runtime/src/startup/start_prototype.rs

```diff
@@ -30,7 +21,7 @@ extern "C" fn start_prototype(
     _app_break: usize,
 ) -> ! {
     use crate::TockSyscalls;
-    use libtock_platform::{RawSyscalls, syscall_class};
+    use libtock_platform::{syscall_class, RawSyscalls};
```

Comment:
Is this right?

Reply:
Yeah, that is a result of running `rustfmt` on this file. I suspect that I never ran `rustfmt` on it before.